### PR TITLE
Navigation feature flags

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-19079

### CODE REVIEW

File: src/js/consts.js [hunk 1]
```diff
@@ -112,6 +115,7 @@ export const visibilityFunctions = {
       return false;
     }
   },
+  featureFlag: checkFeatureFLag,
```
Comment (karelhala) [inline src/js/consts.js:118]: Any reason not to define it here, but rather on line 38?

**Review by karelhala — COMMENTED** [review]
Looking good, just one minor nit pick.

File: src/js/consts.js (hunk 1 above)
Comment (Hyperkid123) [reply src/js/consts.js:118]: I just don't like anonymous functions that's it.

Gotta figure out the test though

**Review by karelhala — APPROVED** [review]

**Comment by Hyperkid123** [comment]
Converting to draft so we don't accordantly include it in the summit build

**Comment by karelhala** [comment]
@Hyperkid123 I think we can mark it as redy for review.

**Review by karelhala — APPROVED** [review]

**Pull request merged**